integration-tests/maven/src/test/java/io/quarkus/maven/it/ListExtensionsIT.java

```diff
@@ -0,0 +1,75 @@
+package io.quarkus.maven.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.commons.io.output.TeeOutputStream;
+import org.apache.maven.shared.invoker.*;
```

Comment:
Seems like this got left over :)

Reply:
Shoot... I was too focussed on `java.io.*`.

Reply:
Done.